source/developer-guides/memory-map.rst

```diff
@@ -1,6 +1,29 @@
 Memory Map
 ------------
 
+Temporary Memory Layout
+^^^^^^^^^^^^^^^^^^^^^^^
+
+|SPN| Stage 1 temporary memory layout::
+
+          Temporary Memory
+  +------------------------------+  Top of Low MEM

```

Comment:
Is this labeling "Top of Low MEM" correct? Or is this 4 GB?

Reply:
You're right. This would be 4 GB. I just forgot what TOLM referred to.